docs/AutocompleteInput.md

```diff
@@ -391,7 +397,7 @@ import { AutocompleteInput, ReferenceInput } from 'react-admin';
 </ReferenceInput>
 ```
 
-In that case, react-admin uses the [`recordRepresentation`](./Resource.md#recordrepresentation) of the related resource to display the record label. In the example above, `<AutocompleteInput>` uses the resource representation of the `authors` resource, which is the `name` property.
```

Comment:
I don't think you should remove this sentence. If it's hard to understand, perhaps you need to add a snippet about the `<Resource>` definition?